modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -160,6 +160,10 @@ public ObjectField addOrUpdateSystemObjectField(
 
 		_validateLabel(labelMap);
 
+		if (Validator.isNotNull(existingObjectField.getRelationshipType())) {
+			_validateRelationship(objectDefinitionId, required);
```

Comment:
This is not supposed to be the objectDefinitionId.

Reply:
Hi @rodrigopaulino 

I don't understand. That is a question? I used the `objectDefinitionId` because the validation needs the id.

Reply:
Hey, @murilostodolni. This wasn't a question. You're passing the an ID, but you can't pass any ID. You have to pass the one it's requiring, which is the object field's.

Reply:
Perfect. It's really wrong. Thanks so much